inputFiles/solidMechanics/ExtendedDruckerPragerWellbore_benchmark.xml

```diff
@@ -10,32 +10,57 @@
     <InternalWellbore
       name="mesh1"
       elementTypes="{ C3D8 }"
-      radius="{ 0.1, 5.0 }"
-      theta="{ 0, 180 }"
+      radius="{ 0.1, 10.0 }"
+      theta="{ 0, 90 }"
       zCoords="{ -1, 1 }"
       nr="{ 40 }"
-      nt="{ 80 }"
-      nz="{ 2 }"
+      nt="{ 40 }"
+      nz="{ 1 }"
       trajectory="{ { 0.0, 0.0, -1.0 },
                     { 0.0, 0.0, 1.0 } }"
       autoSpaceRadialElems="{ 1 }"
-      useCartesianOuterBoundary="0"
```

Comment:
I did not see the reason why switching to radial shaped outer boundary. I prefer to use the Cartesian aligned outer boundary.

Reply:
The shape of the far-field boundary does not really matter in this example. However a cylindrical shape of the far-field boundary provides a more optimized local mesh around the wellbore surface for this axisymmetric problem.